src/EnergyPlus/PluginManager.cc

```diff
@@ -830,7 +830,7 @@ void PluginInstance::setup([[maybe_unused]] EnergyPlusData &state)
         char *zStr = PyBytes_AsString(pStrObj);
         std::string s(zStr);
         Py_DECREF(pStrObj); // PyUnicode_AsUTF8String returns a new reference, decrement it
-        ShowMessage(state, format("PythonPlugin: Class {} imported from: ", className));
+        ShowMessage(state, format("PythonPlugin: Class {} imported from: {}", className, s));
```

Comment:
Good fix.  As small as it is, I think I'll let CI make a full pass over it for fun.  If we find anything on any other platforms, I might just add them to this branch.  Thanks @mitchute